### DISCUSSION TITLE
Update from netbox

### DISCUSSION THREAD

**Discussion by iipetrov** [netbox-community/netbox#12224] [Help Wanted!] [answered]
Hello friends, 
I'm traying to update my netbox version from v3.4.7 to v3.5-beta1 via "sudo git checkout v3.5-beta1"
but I have this error "error: pathspec 'v3.5-beta1' did not match any file(s) known to git". So my netbox has been installed via git so it should not have any issues but .... 
Can you somene help ?

**Comment by a084ed22** [comment, marked as answer]
You may need to run `git fetch --tags` to update your local repository with the remote updated content and the tags

**Reply by iipetrov** [reply]
Thank you, now it's working :)

**Reply by candlerb** [reply]
FYI, just "git fetch" would have done.

Do bear in mind: never upgrade a production instance of Netbox to a beta version. There is no supported upgrade path from a beta version to the final version (in this case v3.5.0 when it comes out), so you'll have to throw away all your data and start with a fresh instance.

What you should do is *clone* your production instance to a test instance, then upgrade the test instance to beta, then test that.

**Reply by iipetrov** [reply]
Thank you. This is really good point!
